src/datepicker/utils/date-utils-provider.js

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import * as React from 'react';
+import DateFnsUtils from '@date-io/date-fns';
+
+const baseInstance = new DateFnsUtils();
+
+export const DateUtilsContext: React.Context<ThemeT> = React.createContext(
+  baseInstance,
```

Comment:
Can this move to a property on the context object? Maybe like `React.createContext({adapter: new DateFnsUtils()});`. Doing so would signal a wrapped api in the code more clearly.

Reply:
Yep, I was going back and forth on this.  It definitely makes it more clear.